packages/fuels-abigen-macro/tests/test_projects/swayswap/swayswap_abi/Forc.toml

```diff
@@ -0,0 +1,5 @@
+[project]
+authors = ["Mohammad"]
```

Comment:
I don't think the `swayswap` project should be in there

Reply:
I believe it should. It's a massive test case for the SDK, it covers a _lot_ of use cases.